diff --git a/server/channels/app/limits.go b/server/channels/app/limits.go
index b131038..8d67c80 100644
--- a/server/channels/app/limits.go
+++ b/server/channels/app/limits.go
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	maxUsersLimit     = 200
-	maxUsersHardLimit = 250
+	maxUsersLimit     = 5000000
+	maxUsersHardLimit = 11000000
 )
 
 func (a *App) GetServerLimits() (*model.ServerLimits, *model.AppError) {
